frontend/simple/controller/router.js

```diff
@@ -209,6 +211,39 @@ var router = new Router({
       },
       beforeEnter: createEnterGuards(loginGuard)
     },
+    {
+      path: '/messages',
+      name: 'Messages',
+      component: Messages,
+      meta: {
+        title: 'Messages'
+      },
+      beforeEnter: createEnterGuards(loginGuard) // NOTE comment this to test on a iPhone
+    },
+    {
+      path: '/messages/:chatName',
+      name: 'MessagesConversation',
+      component: Messages,
+      beforeEnter: createEnterGuards(loginGuard) // NOTE comment this to test on a iPhone
+      // BUG/REVIEW "CANNOT GET /:username" when username has "." in it
+      // ex: messages/joe.kim doesnt work but messages/joekim works fine.
+      // Possible Solution: https://router.vuejs.org/guide/essentials/history-mode.html#example-server-configurations
```

Comment:
If dots in usernames don't work then maybe we need to fix this and make the username as a parameter? Or if you don't want to do that in this PR, can you open an issue for it?

Reply:
I'll create an issue for that :)